### PR TITLE
docs: update issue form templates

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
add checklist with proper validation format

## Related Issue

Closes: #393 

### Describe the changes you've made

```yml
- type: checkboxes
    id: terms
    attributes:
      label: Checklist
      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)
      options:
        - label: "I have read the [Contributing Guidelines](https://github.com/WebXDAO/WebXDAO.github.io/blob/main/CONTRIBUTING.md)"
          required: true
        - label: "I have checked the existing [issues](https://github.com/WebXDAO/WebXDAO.github.io/issues)"
          required: true
        - label: "I am willing to work on this issue (optional)"
          required: false
        - label: "I am a GSSoC'23 contributor"
          required: false
```

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

- There is no error with the file.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original          |      Updated       |
| :-----------------: | :----------------: |
| image | image |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Anmol-Baranwal** [comment]
Kindly add appropriate labels for GSSoC to both the issues and the pull request :)

**Review by vinzvinci — COMMENTED** [review]
Thank you! 

Please wait for our mentors to review this !

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -24,8 +24,14 @@ body:
   - type: checkboxes
     id: terms
     attributes:
-      label: Code of Conduct
+      label: Checklist
       description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)
       options:
-        - label: I agree to follow this project's Code of Conduct
+        - label: "I have read the [Contributing Guidelines](https://github.com/WebXDAO/WebXDAO.github.io/blob/main/CONTRIBUTING.md)"
```
Comment (shuklaritvik06) [inline .github/ISSUE_TEMPLATE/bug_report.yml:30]: It will be rendered as a string not as a markdown

**Review by shuklaritvik06 — APPROVED** [review]

**Review by mkubdev — DISMISSED** [review]
LGTM :)

**Review by mkubdev — APPROVED** [review]
Re LGTM!

**Comment by Anmol-Baranwal** [comment]
@vinzvinci
Please add an appropriate label: There are three labels (levels) to consider for PRs under GSSoC'23, and without them, it would be very difficult to create the accurate final leaderboard with properly allocated points.

**Pull request merged**